scripts/plots/SDMESONET_plot.sh

```
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+. /mesonet/nawips/Gemenviron.profile
```

Shellcheck (0.11.0)

[info] Not following: /mesonet/nawips/Gemenviron.profile was not specified as input (see shellcheck -x).

(SC1091)